apis/infix-en_GB/atrium-api-infix-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/infix/en_GB/IteratorAssertionSpec.kt

```diff
@@ -28,5 +36,6 @@ class IteratorAssertionSpec : ch.tutteli.atrium.specs.integration.IteratorAssert
 
         a1 = a1 has next
         a1 = a1 hasNot next
+        a1 = a1 containsNoDuplicates next
```

Comment:
IMO this does not make sense, maybe change to `contains noDuplicates`?

Reply:
Maybe to remove infix at all for this feature seems like it is not making sense at all, what you think?

Reply:
I think it makes sense, for instance
```
expect(listOf(1,2,2)) all { it isGreaterThan 1 } and contains noDuplicates
```

Reply:
done